policyengine_canada/variables/gov/provinces/qc/tax/income/credits/children_activities/qc_children_activities_credit.py

```diff
@@ -0,0 +1,39 @@
+from policyengine_canada.model_api import *
+
+
+class qc_children_activities_credit(Variable):
+    value_type = float
+    entity = Household
+    label = "Quebec children's activities tax credit"
+    definition_period = YEAR
+    defined_for = ProvinceCode.QC
+
+    def formula(household, period, parameters):
+        p = parameters(
+            period
+        ).gov.provinces.qc.tax.income.credits.children_activities
+
+        # income eligibility
+        income_eligible = (
```

Comment:
this should be included into the eligibility of each disabled and non disabled credit